### DISCUSSION TITLE
Weekly Governance Updates (13 December, 2024)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#1091] [General]
### UPDATED 14 DECEMBER 2024

This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0067 - PoRep Security Policy & Replacement Sealing Enforcement [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0067.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/727)
- FIP0082 - Add support for aggregated replica update proofs[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/752)
- FIP0086 - Fast Finbality in Filecoin (F3) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/809)
- FIP0091: Add support for Homestead and EIP-155 Ethereum Transactions ("legacy" Ethereum Transactions) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0091.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/962)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

* Update FIP0001 [(PR)](https://github.com/filecoin-project/FIPs/pull/850)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/799)
* On-chain Retrieval Expectations [(PR)](https://github.com/filecoin-project/FIPs/pull/862)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/861) 
* FIPXXXX: Export Sector Termination Method for Miner Action [(PR)](https://github.com/filecoin-project/FIPs/pull/1035)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/1034)
* Common Node API [(PR)](https://github.com/filecoin-project/FIPs/pull/1027) 
* FIP0093: Set the Mining Reserve to Zero [(PR)](https://github.com/filecoin-project/FIPs/pull/1039)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/1030)
* Convert Fundraising Remainder Addresses(es) to Keyless Account Actor(s) [(PR)](https://github.com/filecoin-project/FIPs/pull/1080)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/1033) 
* FRC: Retrieval Checking Requirements [(PR)](https://github.com/filecoin-project/FIPs/pull/1089) 

**Last Call** - *Final opportunity for community feedback!* 
 NA

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas model adjustment for user programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316) 
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301) 
- FIP0077 - Add Cost Opportunity For New Miner Creation [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0077.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/780)
- FIP0078 - Remove Restrictions on the Minting of Datacap [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0078.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/844)
- FIP0080 - Phasing Out Fil+ and Restoring Deal Quality Multiplier to 1x [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0080.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/774)
- FIP0096 - Convert fundraising remainder address(es) to keyless account actor(s) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0096.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/1033)
- FIP0097: Add Support for EIP-1153 (Transient Storage) in the FEVM[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/855) 
- FIP0098: Simplify termination fee calculation to a fixed percentage of initial pledge[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/1036)

 **:fire: HOT :fire: Discussion Topics**

- Check out this new discussion on replacing Code CIDs in actors with stable IDs by @rvagg 

---
**Other Governance Updates**
- The scheduled calibration upgrade for nv25 has been canceled. you can read more about this and proposed solutions [here](https://github.com/filecoin-project/core-devs/discussions/183#discussioncomment-11553381).  

---

This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

**Comment by rjan90** [comment]
I think the list of `Accepted FIPs - FIPs awaiting network implementation` needs to be updated a bit:

```
Accepted FIPs - FIPs awaiting network implementation

FIP0067 - PoRep Security Policy & Replacement Sealing Enforcement [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0067.md)https://github.com/filecoin-project/FIPs/discussions/727
FIP0081 - Introduce lower bound for sector initial pledge [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0081.md)https://github.com/filecoin-project/FIPs/discussions/847
FIP0086 - Fast Finbality in Filecoin (F3) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md)https://github.com/filecoin-project/FIPs/discussions/809
FIP0091: Add support for Homestead and EIP-155 Ethereum Transactions ("legacy" Ethereum Transactions) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0091.md)https://github.com/filecoin-project/FIPs/discussions/962
FIP0094 - Add Support for EIP-5656 (MCOPY Opcode) in the FEVM [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0094.md) https://github.com/filecoin-project/FIPs/discussions/1025.
FIP0095 - Add FEVM precompile to fetch beacon digest from chain history https://github.com/filecoin-project/FIPs/discussions/1051. [Until 24 September 2024].
```

- FIP0091 has been implemented and landed in nv23: https://github.com/filecoin-project/core-devs/blob/master/Network%20Upgrades/v23.md
- FIP0081, FIP0094 and FIP0095 has been implemented and landed in nv24: https://github.com/filecoin-project/core-devs/blob/master/Network%20Upgrades/v24.md.

I´m also quite sure FIP0067 should not be on that list (or implementation was deemed not needed?), but I will have to double check with the authors.

**Reply by rjan90** [reply]
[FIP-0082: Add support for aggregated replica update proofs](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0082.md) should get added to the list of `Accepted FIPs - FIPs awaiting network implementation`.

**Reply by rjan90** [reply]
> I´m also quite sure FIP0067 should not be on that list (or implementation was deemed not needed?), but I will have to double check with the authors.

Quick update on the [FIP-0067](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0067.md) from the authors, it is basically a `<break glass in case of emergency> FIP`.
